### PR TITLE
Favlist

### PR DESCRIPTION
**Pull request by SOA-P**

### CODE REVIEW

**Review by hswiergiel — DISMISSED** [review]

File: frontend/src/pages/Offer/Offer.jsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (LoicBrassart) [inline frontend/src/pages/Offer/Offer.jsx]: Remove this

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Stop disabling ESLint plz

**Review by hswiergiel — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Nope, the change I requested was not made

File: frontend/src/pages/Offer/Offer.jsx (hunk 1 above)
Comment (LoicBrassart) [reply frontend/src/pages/Offer/Offer.jsx]: This code is still there

**Review by LoicBrassart — DISMISSED** [review]

**Review by Azahires — APPROVED** [review]

**Review by fantinerudent — APPROVED** [review]

**Pull request merged**